### ISSUE TITLE
Support Mekanism-style gases

### ISSUE THREAD

**Issue by josephcsible** [CyclopsMC/IntegratedDynamics#440]
#### Issue type:

_Only keep the issue types that are applicable._

- :heavy_plus_sign: Feature request

We currently don't have any support at all for gases like Mekanism uses (not gaseous fluids, but their own thing entirely). We should look into adding this.

**Comment by rubensworks** [comment]
I wouldn't consider this urgent at all. Mekanism should make its gases Forge fluids (as it _is_ a fluid after all)

**Comment by josephcsible** [comment]
That's been brought up before, and they've said no: see aidancbrady/Mekanism#4164 and aidancbrady/Mekanism#4528.

**Comment by rubensworks** [comment]
Yeah, I don't think _all other mods_ should provide support for these gases just because Mekanism doesn't adhere to the standards.

**Comment by josephcsible** [comment]
Yeah, but I do kind of see their point: in real life, it's a lot harder to transport a pressurized gas than a liquid (and Mekanism uses this fact for balance), but virtually no mods today bother to check if a fluid is gaseous when they work with it.

**Comment by rubensworks** [comment]
> virtually no mods today bother to check if a fluid is gaseous when they work with it

Then this would be something that other mods should support. That's probably a lot less effort than adding support for a completely separate gases API.

**Comment by cauamachado51** [comment]
up

**Comment by rubensworks** [comment]
This can be closed, now that we have https://github.com/CyclopsMC/IntegratedMekanism
(This new mod is being released on CurseForge and Modrinth as I am writing this)

<blockquote><img src="https://opengraph.githubassets.com/e24209f7fd41cd3804c544bfdd370b98ad52bd88b4e2dea92a048dde481591bb/CyclopsMC/IntegratedMekanism" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/IntegratedMekanism">GitHub - CyclopsMC/IntegratedMekanism: Compatibility mod between Mekanism and the Integrated Dynamics mods</a></strong></div><div>Compatibility mod between Mekanism and the Integrated Dynamics mods - CyclopsMC/IntegratedMekanism</div></blockquote>
